src/html/undefined-elements.ts

```diff
@@ -41,15 +41,21 @@ export class UndefinedElements extends HtmlRule {
         // HACK. Filed as https://github.com/Polymer/polymer-analyzer/issues/507
         continue;
       }
+      if (!ref.astNode) {
+        continue;
+      }
       const el = document.getById(
           'element', ref.tagName, {imported: true, externalPackages: true});
 
       if (el.size === 0) {
+        const sourceRange = parsedDocument.sourceRangeForStartTag(ref.astNode);
+        if (!sourceRange) {
```

Comment:
It's a bit sloppy to have these unrelated changes here, but I was seeing some weird errors from these occasionally being undefined.